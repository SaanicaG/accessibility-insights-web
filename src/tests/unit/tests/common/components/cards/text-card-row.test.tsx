// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.
import { render } from '@testing-library/react';
import { StringPropertyCardRowProps } from 'common/components/cards/get-labelled-string-property-card-row';
import { TextCardRow } from 'common/components/cards/text-card-row';
import { CardRowDeps } from 'common/configs/unified-result-property-configurations';
import * as React from 'react';

describe('TextCardRow', () => {
    it('renders', () => {
        const props: StringPropertyCardRowProps = {
            propertyData: 'test text',
            deps: {} as CardRowDeps,
            index: -1,
        };

        const renderResult = render(<TextCardRow {...props} />);
        expect(renderResult.asFragment()).toMatchSnapshot();
    });
});
