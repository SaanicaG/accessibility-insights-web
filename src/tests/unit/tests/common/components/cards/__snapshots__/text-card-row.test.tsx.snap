// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextCardRow renders 1`] = `
<DocumentFragment>
  Texttest text
</DocumentFragment>
`;
